test/fixtures/poro.rb

```diff
@@ -1,3 +1,4 @@
+ActiveModelSerializers::Model.attributes_are_always_the_initialization_data = false
```

Comment:
see https://github.com/rails-api/active_model_serializers/pull/1998/files#r93893962 but the test poro superclass can't have the legacy behavior included because there's no simple way to remove it.  Instead, the test poro superclass has the 'new' behavior, but, subclasses of it, by default, will have the 'old' behavior.  Basically, it kind of extends the AMS::Model in the test context so that we can test it with the legacy behavior on and off.